transient-extras.el

```diff
@@ -0,0 +1,115 @@
+;; transient-extras.el --- Extra features for transient -*- lexical-binding: t -*-
+
+;; TODO: Package Header
+
+;; TODO: License information
+
+;;; Commentary:
+
+(require 'transient)
+
+;;; Code:
+
+
+;;; Files Lists
+
+;; TODO: (Planned feature) A way to add/remove files instead of replacing.
+;; For example by pressing:
+;; `-+': add files
+;; `-=': remove files
+;; `--': replace files
```

Comment:
This probably should be removed since I copied it to as Github issue.

Reply:
Will do